### DISCUSSION TITLE
Waypoint not appearing in Hubs after being created in Spoke

### DISCUSSION THREAD

**Discussion by github-user-77** [mozilla/hubs#3266] [Spoke Questions] [answered]
Hi all, are there any docs on the waypoint element? I've placed one in my scene, but it is not appearing in Hubs, regardless of the options I choose for it. I assume they are supposed to have a 'click to teleport' type functionality?

**Comment by j-conrad** [comment, marked as answer]
If you have checked the 'Clickable' option on a Waypoint in Spoke, you will be able to do so by first holding Spacebar (or A on an Oculus controller) in Hubs to see it. You can also toggle those items in Hubs with the TAB key.

Here is a video tutorial series on the topic:
https://www.youtube.com/playlist?list=PLCxaiaRxTL69cYpASBeIJBUtJxoiykhk3

**Reply by github-user-77** [reply]
Thanks very much!
